c_emulator/riscv_sim.cpp

```diff
@@ -430,9 +432,9 @@ void init_sail_reset_vector(uint64_t entry)
   zPC = rom_base;
 }
 
-void init_sail(uint64_t elf_entry)
+void init_sail(uint64_t elf_entry, const char *config_file)
 {
-  zinit_model(UNIT);
+  zinit_model(config_file ? config_file : "");
```

Comment:
```suggestion
  zinit_model(config_file != nullptr ? config_file : "");
```

(I generally try to avoid implicit bool conversions - even though C++ lets you do it it's bad style.)